skactiveml/utils/tests/test_visualisation.py

```diff
@@ -0,0 +1,41 @@
+import unittest
```

Comment:
@LukasLuehrs Adjust test according to changes proposed in `skactiveml.utils._visualisation.py`.